### PR TITLE
Allow images of any size as profile pictures

### PR DESCRIPTION
**Pull request by marcoambrosini**
This pr scales the images to 512px in size or keeps them the same size if they're smaller than that. This allows us to remove the limit warning.

### CODE REVIEW

**Review by Pytal — APPROVED** [review]

File: apps/settings/src/components/PersonalInfo/AvatarSection.vue [hunk 1]
```diff
@@ -60,7 +60,7 @@
 						</template>
 					</NcButton>
 				</div>
-				<span>{{ t('settings', 'png or jpg, max. 20 MB') }}</span>
+				<span>{{ t('settings', 'The file should be a png or jpg') }}</span>
```
Comment (szaimen) [inline apps/settings/src/components/PersonalInfo/AvatarSection.vue:63]:
```suggestion
				<span>{{ t('settings', 'The file must be a png or jpg') }}</span>
```
Maybe?

**Review by szaimen — APPROVED** [review]
Seems to work in my testing :)

**Review by rakekniven — DISMISSED** [review]

**Review by artonge — APPROVED** [review]

**Comment by szaimen** [comment]
@marcoambrosini could you have a look at the cypress test?

**Comment by nickvergessen** [comment]
> could you have a look at the cypress test?

The files version one? Looks very unrelated. Maybe a ticket can be created instead?

**Comment by szaimen** [comment]
> The files version one? Looks very unrelated.

indeed, looks like it fails on master as well...

**Comment by szaimen** [comment]
So merge then?

**Pull request merged**